### ISSUE TITLE
Beartype's Type signature breaks some features of pylance

### ISSUE THREAD

**Issue by jonathanmorley** [beartype/beartype#36]
Hovering over an `@beartype` annotated function in VSCode with the Python plugin (using Pylance) shows the function as

```
(method) foo: (*args: Unknown, **kwargs: Unknown) -> Unknown
```

This can be reproduced with a simple decorator function using the same type signature as `beartype`:

```
def beartype2(func: Callable) -> Callable:
    return func
```

**Comment by leycec** [comment]
Gratuitous type variables for the win! Your solution is both perfect and clever. :smiley_cat: 

I *really* appreciate the hard work you invested here. @beartype is now that much more usable for devs using bonafide IDEs. <sup>so... everybody but me</sup> Thanks again, Maestro Morley.

**Comment by leycec** [comment]
@jonathanmorley: `beartype 0.7.1` has now been released, complete with your miraculous fix at #37. Thanks again for the generous donation of your free time and intellect. :brain:
